great_expectations/render/renderer/site_builder.py

```diff
@@ -410,6 +424,9 @@ def build(self):
         resource_keys = target_store.list_keys()
         index_links_dict = OrderedDict()
 
+        if self.show_cta_footer:
+            index_links_dict["cta_object"] = self.get_cta_object()
```

Comment:
`index_links_dict` has a new `cta_object` key that index page renderer uses to render cta footer if present